src/app/services/sort-helper.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable, Type } from '@angular/core';
+import { Sort } from '../models/sort';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SortHelperService {
+
```

Comment:
I dont't think you need a service for sorting. This could be just some helper functions